src/botPage/view/logger.js

```diff
@@ -37,6 +37,11 @@ const isNewError = isNewMessage();
 const notify = ({ className, message, position = 'left', sound = 'silent' }) => {
     if (message && (position === 'left' || isNewNotification(message))) {
         log(className, message);
+        if (typeof message === 'number') {
+            // eslint-disable-next-line no-param-reassign
+            message = message.toFixed(4);
```

Comment:
This will show every number passed to the notification block with 4 decimals, not the behaviour we want.